### PR TITLE
Require clear descriptions both for feature and breaking PRs prior to the merge

### PR DESCRIPTION
**Pull request by delvh**
When writing the release blog, it is really annoying and time-consuming to re-discover and write down how a feature behaves and capture a screenshot of it, for every single feature merged since the last release.
This should not be the responsibility of maintainers, but rather of the person implementing a feature in the first place.
They know best how to use the feature and how to gather screenshots for it.
Similarly for breaking changes and their effects.

As such, let's require everything to be up-to-date and easily understandable before merging features or breaking changes.

### CODE REVIEW

**Review by lafriks — APPROVED** [review]

**Review by denyskon — APPROVED** [review]
I'd also like to make correct labels a requirement before merging, but that's a responsibility of the corresponding maintainers, not the PR author.

**Comment by delvh** [comment]
Should I add it to this PR?

**Comment by delvh** [comment]
> Another requirement for merging PRs is that the PR is labeled correctly.\
However, this is not your job as a contributor, but the job of the person merging your PR.

How about that?

**Comment by denyskon** [comment]
Sounds good. Maybe add:

> If you think that your PR was labeled incorrectly, or notice that it was merged without labels, please let us know.

**Review by yardenshoham — APPROVED** [review]

**Comment by delvh** [comment]
Labels are now required to be correct too.

**Review by denyskon — APPROVED** [review]

**Review by 6543 — APPROVED** [review]

**Pull request merged**
